packages/frontend/src/components/common/FieldSelect/FieldSelect.tsx

```diff
@@ -1,18 +1,18 @@
 import { Field, getItemId, TableCalculation } from '@lightdash/common';
-import { Select } from '@mantine/core';
+import { Select, SelectProps } from '@mantine/core';
 import { FC } from 'react';
-import FieldIcon from '../../common/Filters/FieldIcon';
-import { fieldLabelText } from '../../common/Filters/FieldLabel';
-import FieldSelectItem from '../FieldSelectItem';
+import FieldIcon from '../Filters/FieldIcon';
+import { fieldLabelText } from '../Filters/FieldLabel';
+import FieldSelectItem from './FieldSelectItem';
 
-interface Props {
+type Props = Omit<SelectProps, 'data'> & {
     label?: string;
     selectedField?: Field | TableCalculation;
     fieldOptions: (Field | TableCalculation)[];
     placeholder?: string;
     disabled?: boolean;
```

Comment:
can remove these if we're getting `SelectProps`

Reply:
maybe same for `onChange`?

Reply:
Good call